src/Umbraco.Web.UI/config/log4net.Release.config

```diff
@@ -1,17 +1,19 @@
 <?xml version="1.0"?>
 <log4net>
-  
+
   <root>
     <priority value="Info"/>
     <appender-ref ref="AsynchronousLog4NetAppender" />
   </root>
 
   <appender name="rollingFile" type="log4net.Appender.RollingFileAppender">
-	  <file type="log4net.Util.PatternString" value="App_Data\Logs\UmbracoTraceLog.%property{log4net:HostName}.txt" />
+    <file type="log4net.Util.PatternString" value="App_Data\Logs\UmbracoTraceLog.%property{log4net:HostName}.txt" />
     <lockingModel type="log4net.Appender.FileAppender+MinimalLock" />
     <appendToFile value="true" />
-    <rollingStyle value="Date" />
+    <rollingStyle value="Composite" />
+    <datePattern value=".yyyy-MM-dd.'log'"/>

```

Comment:
I think we'll change this to .txt just to be consistent with the main file, otherwise this is perfect, will pull it in once I test diplo log viewer